tests/integration/events/test_database_duration.php

```diff
@@ -49,7 +49,7 @@ function test_slow_sql() {
   global $PDO_MYSQL_DSN, $MYSQL_USER, $MYSQL_PASSWD;
 
   $conn = new PDO($PDO_MYSQL_DSN, $MYSQL_USER, $MYSQL_PASSWD);
-  $result = $conn->query('select * from tables limit 1;');
+  $result = $conn->query('select * from information_schema.tables limit 1;');
```

Comment:
The changes above to `config.php` make it possible to override `MYSQL_DB` but it seems like the tests all hard code it to `information_schema` - what purpose does the override mechanism serve in this case?

Reply:
After discussion the hard coding of `information_schema` is OK as it is a standard reference in all relation databases we care about, and the override is used for other tests where the value comes from the env var.